aws/sdk/build.gradle.kts

```diff
@@ -266,8 +268,8 @@ task("finalizeSdk") {
         "relocateRuntime",
         "relocateAwsRuntime",
         "relocateExamples",
-        "generateDocs",
-        "fixManifests"
+        "generateDocs" // ,
+        // "fixManifests"
```

Comment:
It produces a good amount of noise doesn't it? I noticed that too when working on the `impl Into` thing

Reply:
it also makes stuff really slow. I need to back this change out but we should find a way to avoid running fixManifests locally